### PR TITLE
Generation of a vector database

### PR DESCRIPTION
**Pull request by AnnaKelmanson**

### CODE REVIEW

File: src/rambo/vdb_generation/8_SEPT_APPROVED_full_dataset.csv
Comment (mlederbauer) [inline src/rambo/vdb_generation/8_SEPT_APPROVED_full_dataset.csv:1]: I would not put the raw dataset here in the source code, maybe point to it in the ./data repo which is in .gitignore?

Comment (mlederbauer) [inline src/rambo/vdb_generation/8_SEPT_APPROVED_full_dataset.csv:1]: I would not put the raw dataset here in the source code, maybe point to it in the ./data repo which is in .gitignore?

Comment (mlederbauer) [reply src/rambo/vdb_generation/8_SEPT_APPROVED_full_dataset.csv:1]: and delete the file in the repo here

**Review by mlederbauer — CHANGES_REQUESTED** [review]

**Comment by doncamilom** [comment]
should we merge this now @mlederbauer ?

**Comment by mlederbauer** [comment]
sounds good @doncamilom  thanks to anna for reviewing and adding the code!

**Pull request merged**